sklearn/tree/tree.py

```diff
@@ -454,6 +454,30 @@ def apply(self, X, check_input=True):
         X = self._validate_X_predict(X, check_input)
         return self.tree_.apply(X)
 
+    def decision_paths(self, X, check_input=True):
```

Comment:
I think this should be `decision_path`, not `decision_paths`. I don't like the plural. Minor comment, though.